packages/core/src/domain/configuration.ts

```diff
@@ -1,9 +1,16 @@
 import { BuildEnv } from '../boot/init'
 import { CookieOptions, getCurrentSite } from '../browser/cookie'
 import { catchUserErrors } from '../tools/catchUserErrors'
-import { includes, ONE_KILO_BYTE, ONE_SECOND } from '../tools/utils'
+import { includes, objectValues, ONE_KILO_BYTE, ONE_SECOND } from '../tools/utils'
 import { computeTransportConfiguration } from './transportConfiguration'
 
+export const InitialPrivacyLevel = {
+  ALLOW: 'allow',
+  MASK: 'mask',
+  MASK_FORMS_ONLY: 'mask-forms-only',
+} as const
+export type InitialPrivacyLevel = typeof InitialPrivacyLevel[keyof typeof InitialPrivacyLevel]
```

Comment:
We chose to avoid exposing enums publicly especially to support CDN setups. We want to let the customer use plain values (ex: `'allow'`) so they don't need to import them (ex: in CDN setups importing stuff is not ideal). Also, when using TS definitions + a CDN setup like we do on web-ui, we can't import the enum value with`import type { SomeEnum } from "browser-rum"`. Because we cannot use this specific enum, typecheck fails. (Note: we can't do `import { SomeEnum } from  "browser-rum"` because our package have side effects that would be triggered by importing it)